### ISSUE TITLE
Idea for making playlists and watch history work

### ISSUE THREAD

**Issue by JustCallMeQuincy** [meeeeemu/youtube-antiadblock-block#1]
Hi! This is a small idea I had after my YouTube bugged out a bit.

Basically, the video I was watching had embedded, as expected.
When I sent a tab from my phone, then the tab went asleep and got woken up again, the embedded video went back to the homepage **INSIDE OF THE EMBED**.

YouTube now basically ran from inside of an embed.

This fixes the popup (since it's an embed) and it behaves like regular YouTube, apart from fullscreen not being available.

*The YouTube page running inside of the other YouTube video's embed*
![image](https://github.com/meeeeemu/youtube-antiadblock-block/assets/71931404/ff21cbbc-26bf-47f5-b022-4a7e46cfce04)

### TL;DR - Instead of *just* a video running in an embed, we run the entire YouTube page in an embed

**Comment by JustCallMeQuincy** [comment]
I hope this is somewhat helpful in some way, if it's not, feel free to close this issue

**Comment by JustCallMeQuincy** [comment]
@meeeeemu (little bump, just in case you're not subscribed to the repo)

**Comment by meeeeemu** [comment]
this is an interesting discovery! the main issue i think might occur though is that users would probably get logged out each time they visited youtube (maybe) i’d ideally want to replace the entire website with an embed, which should still block the adblock thing. i’d have to scrap the current solution or maybe let people pick a mode of some sorts??? i dunno

i’ll close this issue for now and see what i can do with it in the future. appreciate the idea :)
